grails-app/controllers/org/pih/warehouse/api/StocklistItemApiController.groovy

```diff
@@ -70,7 +70,7 @@ class StocklistItemApiController {
         render([data: stocklistItem] as JSON)
     }
 
-    def delete = {
+    def deleteItem = {
```

Comment:
I am not sure it this is proper approach. @jmiranda is it ok for you?

Reply:
@awalkowiak yes, this shouldn't be necessary.

@DominikaHatala can you explain why you think you need to do this? what error or behavior were you seeing?

Reply:
@jmiranda I have responded in ticket's comments.